controller/setting_controller.go

```diff
@@ -390,6 +395,85 @@ func (sc *SettingController) syncBackupTarget() (err error) {
 	return nil
 }
 
+func (sc *SettingController) handleSecretsForAWSIAMRoleAnnotation(backupTargetURL, oldSecretName, newSecretName string, isBackupTargetURLChanged bool) (err error) {
+	isSameSecretName := oldSecretName == newSecretName
+	if isSameSecretName && !isBackupTargetURLChanged {
+		return nil
+	}
+
+	isHavingAWSIAMRoleArn := false
```

Comment:
nit: consider using `isArnExists` instead to improve readability.

Reply:
Modified to `isArnExists`.Thanks